commit e80225d36dca7c1897dc3b6107ee9ee87fafc9a4
Author: Eric Anholt <eric@anholt.net>
Date:   Fri Mar 16 15:04:34 2018 -0700

    drm/vc4: Add missing formats to vc4_format_mod_supported().
    
    [ Upstream commit 1e871d65e375280757833d9fce91dda71980bdf5 ]
    
    Daniel's format_mod_supported() patch predated Dave's for NV21/61, and
    I didn't catch that when rebasing.  This is a problem since the
    formats are now getting validated before being passed to the driver's
    atomic hooks.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>
    Acked-by: Daniel Stone <daniels@collabora.com>
    Cc: Dave Stevenson <dave.stevenson@raspberrypi.org>
    Fixes: 423ad7b3cbd1 ("drm/vc4: Advertise supported modifiers for planes")
    Link: https://patchwork.freedesktop.org/patch/msgid/20180316220435.31416-2-eric@anholt.net
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/vc4/vc4_plane.c b/drivers/gpu/drm/vc4/vc4_plane.c
index cf5aea1d6488..3363451b992b 100644
--- a/drivers/gpu/drm/vc4/vc4_plane.c
+++ b/drivers/gpu/drm/vc4/vc4_plane.c
@@ -874,7 +874,9 @@ static bool vc4_format_mod_supported(struct drm_plane *plane,
 	case DRM_FORMAT_YUV420:
 	case DRM_FORMAT_YVU420:
 	case DRM_FORMAT_NV12:
+	case DRM_FORMAT_NV21:
 	case DRM_FORMAT_NV16:
+	case DRM_FORMAT_NV61:
 	default:
 		return (modifier == DRM_FORMAT_MOD_LINEAR);
 	}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1215_e80225d36dca7c1897dc3b6107ee9ee87fafc9a4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1215_e80225d36dca7c1897dc3b6107ee9ee87fafc9a4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
